kratos/python_scripts/time_based_ascii_file_writer_utility.py

```diff
@@ -129,40 +135,10 @@ def __ValidateAndAssignOutputFolderPath(self):
         if not self.file_name.endswith(".dat"):
             self.file_name += ".dat"
 
-        # check if relative path was erroneously specified in file name
-        raw_path, raw_file_name = os.path.split(self.file_name)
+        _, raw_file_name = os.path.split(self.file_name)
```

Comment:
can you add a deprecation warning in case the first argument is not empty? => this basically says that folders can only be specified through the `output_path` argument

Reply:
Ok.